app/src/main/java/com/example/peterchu/watplanner/Models/Schedule/CourseComponent.java

```diff
@@ -281,15 +281,15 @@ public Calendar getCalendarEndTime() {
 
                     event.setStartTime(eventStartDate);
                     event.setEndTime(eventEndDate);
-                    event.setLocation(location.toString());
+                    event.setLocation(type);
```

Comment:
so we're setting the location field to show the type? don't we want to show the location too?